test('hello world', ()=>{

})

test('failed', ()=>{
    throw new error('failed')
})